src/theme/_typography.scss

```diff
@@ -243,6 +243,7 @@ p {
   font-size: var(--font-size-m);
   text-decoration: underline;
   cursor: pointer;
+  font-family: var(--font-secondary);
```

Comment:
Zijn alle links in het design secondary font?

Reply:
Ik denk het wel

Reply:
Is ok dan!